### PR TITLE
Move SendFeedback into its own component

### PR DESCRIPTION
**Pull request by mirshko**
since the feedback components single responsibility is to send an email and doesn't care about the context of where the button is it makes sense for it to live on its own rather than passing the mailer function down into the settings screen

### CODE REVIEW

**Comment by ricburton** [comment]
https://media.giphy.com/media/swtiK9jRfE0zS/giphy.gif

File: src/components/SendFeedback.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { debounce } from "lodash";
+import React from "react";
+import { Alert, Clipboard } from "react-native";
+import Mailer from "react-native-mail";
+
```
Comment (mikedemarais) [inline src/components/SendFeedback.js]: delete line!

File: src/screens/SettingsScreen.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { withHandlers } from 'recompact';
 import styled from 'styled-components/primitives';
+
```
Comment (mikedemarais) [inline src/screens/SettingsScreen.js]: delete blank line!

File: src/screens/SettingsScreen.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import PropTypes from 'prop-types';
...
+import SendFeedback from "../components/SendFeedback";
```
Comment (mikedemarais) [inline src/screens/SettingsScreen.js]: move to line 10 to be `abc` alphabetical

File: src/screens/SettingsScreenWithData.js [hunk 4]
```diff
@@ -1,24 +1,11 @@
-import { debounce, omit } from 'lodash';
+import { omit } from 'lodash';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { Alert, Clipboard } from 'react-native';
-import Mailer from 'react-native-mail';
+
```
Comment (mikedemarais) [inline src/screens/SettingsScreenWithData.js]: delete blank line!

File: src/screens/SettingsScreen.js (hunk 3 above)
Comment (mirshko) [reply src/screens/SettingsScreen.js]: 👍

**Comment by mirshko** [comment]
@mikedemarais on the delete lines comment. i've often seen using a line break to separate imported local components from components / functions imported from node_packages. just as a way to keep them two visually separate. thoughts?

**Comment by mikedemarais** [comment]
@mirshko yeah, i know what you are talking about.. this is really one of those super subjective preferences for sure.. 

my preference is: 
npm imports - sorted abc
local imports - sorted by descending depth of relative path, then sorted abc

example of this: 
```
import PropTypes from 'prop-types';
import React from 'react';
import QRCode from 'react-native-qrcode-svg';
import styled from 'styled-components/primitives';
import { Centered } from '../components/layout';
import { colors, padding, position, shadow } from '../styles';
import Button from './Button';
```

**Pull request merged**